rocketpy/Rocket.py

```diff
@@ -579,20 +590,93 @@ def addFins(
         Ymac = (
```

Comment:
I could not find this definition on Barrowman's equations, do you remember where did you find?

Not saying it's wrong, but I didn't understand

Reply:
The definition of `Ymac`? I think the 'mac' subtext is from OpenRocket's documentation. I'll change it to `Yma` which is the form used on Barrowman to make the code more consistent.
But here, it is on equation 3-26 on Barrowman:
![image](https://user-images.githubusercontent.com/69485049/153712225-aabacba2-890c-40b4-8d10-4697efef31cb.png)